Source/Core/DolphinQt/Config/Mapping/MappingIndicator.cpp

```diff
@@ -301,6 +326,19 @@ void ReshapableInputIndicator::DrawReshapableInput(
   p.drawPolygon(
       GetPolygonFromRadiusGetter([&stick](double ang) { return stick.GetGateRadiusAtAngle(ang); }));
 
+  const double size = stick.GetVirtualNotchDegrees() * (MathUtil::TAU / 360);
```

Comment:
This added block should likely be placed into its own function and be called here.